src/05-instruction-experiments/00-prompt-lib/collector-profile-one.ts

```diff
@@ -0,0 +1,56 @@
+
+Your goal in this step is to determine the budget of the art collector. You will call get_user_profile and deterine if the response includes the collectors budget. If it does, present the budget to the collector and ask if it is correct. If they say it is not accurate, ask them to provide the desired budget. Once they provide a valid budget, thank them for the information and then continue to step 4.
+
+Your goal in this step is to create a defition of the collectors taste in art. Call get_user_profile and evaluate the user's follows, saves and genes from artsy.net. A genes are characteristics that connect artists, artworks, architecture, and design objects across history. DO NOT use the word gene or genes in your response to the collector. The defition of the collectors taste in art should be no longer than four sentances. Suggest a definition of the collectors taste in art to the user. Confirm with the user that the suggested definition is accurate. If they say it is not accurate, ask them what could be improved and suggest a new definition based on their response. Ask them to confirm that the new suggestion is accurate. Once you have a confirmed the definition is accurate, continue to step 5.
+
+Your goal in this step is to confirm that the user is happy witht he profile that you have created for them. Present the user with the complete profile and ask them to confirm that it is accurate. If they say it is not accurate, ask them what could be improved and suggest a new definition based on their response. Ask them to confirm that the new suggestion is accurate. Once you have a confirmed the definition is accurate. Once the user is happy with the profile, thank them for their time and end the conversation.`
+
+/*
+NOTES:
+
+- It always asks me to confirm I want to build a user profile. 
+- Does seem to do a fairly good job when getting some basic information from the user profile.See https://artsy.slack.com/archives/C06SSV1K10D/p1714418303277419
+- Works when the user doesn't have a profile...though, would this be a use case we need to support?
+- When there is no user profile, it asks for all the information at one time. While we may not ever ask for information from someone who is not a artsy user, we should be aware that a lack of information may result in a question dump. 
+- I got much more specific questions when using the advisor without a user profile. 
+
+
+
+Things noticed
+  - "...with a generous budget of up to $200,000", it should not include language like "generous". This does seem to be solved, when it was asked to change to the first person, though.
+  - Super awkward: "I collect art both for investment purposes and for aesthetic enjoyment, which reflects my appreciation for both the intrinsic and financial value of art."
```

Comment:
Eeek, LLM serving up cold hard truths 😅  

In this instance, did that statement seem grounded in the user input at all?

Reply:
Yeah, I should have been more explicit about the circumstances that resulting in this. It was when I was unauthenticated, and it asked me directly what my purposed for collecting was. Something like "Do you collect for aesthetic enjoyment, investment opportunities, or a mix of both?". So in that sense it was accurate...but something we would want to guard against.